capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -95,6 +97,16 @@ class ClubPage extends Component {
         });
   }
 
+  handleBookChange = async (gbookID) => {
+    this.state.club.gbookID = gbookID;
```

Comment:
Use `setState` instead of modifying `state` directly (https://reactjs.org/docs/state-and-lifecycle.html#do-not-modify-state-directly).

Reply:
This change is from a currently un-merged PR. Once that's merged in this will be resolved.